site/globals/langs.json

```
@@ -83,5 +83,10 @@
         "code": "ja",
         "url": "/ja/",
         "name": "日本語"
+    },
+        {
+        "code": "ua",
+        "url": "/ua/",
```

```suggestion
        "code": "uk",
        "url": "/uk/",
```
згідно списку [мовних кодів ISO](https://uk.wikipedia.org/wiki/%D0%A1%D0%BF%D0%B8%D1%81%D0%BE%D0%BA_%D0%BC%D0%BE%D0%B2%D0%BD%D0%B8%D1%85_%D0%BA%D0%BE%D0%B4%D1%96%D0%B2_ISO_639)